docs/content/preview/faq/yugabytedb-managed-faq.md

```diff
@@ -150,11 +150,12 @@ YugabyteDB Managed uses a shared responsibility model for security. For more inf
 
 ### What cluster configurations can I create?
 
-Using YugabyteDB Managed, you can create single region clusters that can be deployed across multiple and single availability zones.
+Using YugabyteDB Managed, you can create single- and multi-region clusters that can be deployed across multiple and single availability zones.
 
-The Fault Tolerance of a cluster determines how resilient the cluster is to node and availability zone failures and, by extension, the cluster configuration. You can configure clusters with the following fault tolerances in YugabyteDB Managed:
+The Fault Tolerance of a cluster determines how resilient the cluster is to failures and, by extension, the cluster configuration. You can configure clusters with the following fault tolerances in YugabyteDB Managed:
 
-- **Availability Zone Level** - a minimum of 3 nodes spread across multiple availability zones with a RF of 3. YugabyteDB can continue to do reads and writes even in case of an availability zone failure. This configuration provides the maximum protection for a data center failure. Recommended for production deployments. For horizontal scaling, nodes are scaled in increments of 3.
+- **Region Level** - a minimum of 3 nodes spread across 3 regions with a RF of 3. YugabyteDB can continue to do reads and writes even in case of an region failure. This configuration provides the maximum protection for a region failure. Recommended for production deployments. For horizontal scaling, nodes are scaled in increments of 3.
```

Comment:
Is there a reason we call out "recommended for production deployments" ? Setting the FT is based on their application needs. Nothing wrong with running node-level FT if they do not require anything greater.

Reply:
I saw a below edit that has this phrased as "For production clusters, a minimum of Availability Zone Level is recommended. Whether you choose Region or Availability Zone Level depends on your application architecture, design, and latency requirements." in the create-clusters-overview section. This type of wording at the end of the first sentence may be less confusing than included "recommended for production deployments" in the list